### ISSUE TITLE
build fails on newer java

### ISSUE THREAD

**Issue by bwbohl** [music-encoding/music-encoding#993]
In detail it is the `canonicalize-source` and `build-guidelines-html` that fail with:

```
java.lang.UnsupportedOperationException: The Security Manager is deprecated and will be removed in a future release
```

While `build-guidelines-html` already has `@failonerror="true"` and thus makes the whole build fail, `canonicalize-source` would still signal `BUILD SUCCESSFUL` although the canonicalization actually failed.

Adding `@fork="true`on both java-calls and `@failonerror="true"` on `canonicalize-source` will do the trick ;-)

**Comment by musicEnfanthen** [comment]
@bwbohl Thanks for tracking this down. Will this still work with older java versions? Would you like to open a PR?

**Comment by bwbohl** [comment]
fixed in pdf-guidelines-test by 2493df814624861601c8c2386eb05feff8b6d504

**Comment by bwbohl** [comment]
> @bwbohl Thanks for tracking this down. Will this still work with older java versions? Would you like to open a PR?

yes, does work with older java versions. Tested in container with JRE8
